Overview.bs

```diff
@@ -1937,6 +1946,11 @@ Some example inputs and the corresponding expansions:
     <td>14,000,000</td>
     <td>//foo.bar/1Z-A</td>
   </tr>
```

Comment:
Should we use some means of indicating if the key is a string or an integer? Maybe quotes around the strings?

Reply:
Good idea, I added a type column to the example table so there's no confusion.